### ISSUE TITLE
Control arrows missing with linear navigation mode

### ISSUE THREAD

**Issue by rbeezer** [hakimel/reveal.js#2582]
Seems that when `navigationMode` is set to`linear` the last vertical segment has no arrow to access the slides of that segment.  These slides are accessible with the space bar, so perhaps the necessary right arrow is missing?

Problem is symmetric - using the left arrow to work back to the start is fine until the last slide of the first vertical segment, and then there is no left arrow to go back through the slides of the first segment.

MWE attached as a zip file. Tested at tag/release 3.9.2 on Firefox 72.0.2 on Ubuntu 18.04.  Switching to `default` seems to produce correct behavior for that navigation mode.

[linear-order.zip](https://github.com/hakimel/reveal.js/files/4189170/linear-order.zip)

**Comment by earboxer** [comment]
To test this without the above zip file:

```html
<section>
  <section>Slide 1</section>
  <section>Slide 2</section>
</section>
<section>
  <section>Slide 3</section>
  <section>Slide 4</section>
</section>
```

When you are on slide 2, there is no left arrow button (only a right arrow).

When you are on slide 3, there is no right arrow button (only a left arrow).

See https://github.com/hakimel/reveal.js/pull/2416#issuecomment-500294231 (https://github.com/hakimel/reveal.js/commit/bba760e5491f9a8adf3ca239c7eda3423f9b602a).

(When we fix this, we have to make sure it doesn't break vertical-only presentations).

**Comment by hakimel** [comment]
This fix for this has been merged into dev and will be available in 4.0

**Comment by rbeezer** [comment]
Thanks for the quick fix!  We are building reveal.js output from PreTeXt source, so I'll put a note on our tracker to make sure we get caught up once 4.0 is available.
